### PR TITLE
Add k8s-hub deps batch 1

### PR DESCRIPTION
**Pull request by Dentrax**
Fixes:

Related:

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [X] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [X] REQUIRED - The version of the package is still receiving security updates
- [ ] This PR links to the upstream project's support policy (e.g. `endoflife.date`)

#### For new version streams

- [ ] The upstream project actually supports multiple concurrent versions.
- [ ] Any subpackages include the version string in their package name (e.g. `name: ${{package.name}}-compat`)
- [ ] The package (and subpackages) `provides:` logical unversioned forms of the package (e.g. `nodejs`, `nodejs-lts`)

#### For security-related PRs

- [ ] The security fix is recorded in the [advisories](https://github.com/wolfi-dev/advisories) repo

#### For version bump PRs

- [ ] The `epoch` field is reset to 0

#### For PRs that add patches

- [ ] Patch source is documented

### CODE REVIEW

File: py3-jupyterhub-idle-culler.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Generated from https://pypi.org/project/jupyterhub-idle-culler/
+package:
+  name: py3-jupyterhub-idle-culler
+  version: 1.2.1
+  epoch: 0
+  copyright:
+    - license: 3-BSD
```
Comment (mamccorm) [inline py3-jupyterhub-idle-culler.yaml:7]: Is this correct? Doesn't look like an abbreviation we've used before and I don't see listed in: https://opensource.org

File: py3-jupyterhub-idle-culler.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      - busybox
+      - build-base
+      - python-3
+      - py3-setuptools
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: d80822982d2590cb876d01849cee028943fdbdcc991c0f02fce45af3f40dd415
+      uri: https://files.pythonhosted.org/packages/e8/be/f225c5d4cd186f793e46ababecee6ee62b2a57906a0ec1fc0453cda6c6ed/jupyterhub-idle-culler-${{package.version}}.tar.gz
```
Comment (rawlingsj) [inline py3-jupyterhub-idle-culler.yaml:28]: I think melange has been updated recently to default to `git-checkout`

Comment (mamccorm) [inline py3-jupyterhub-idle-culler.yaml:28]: Is there a reason we're not pulling this down from git?
https://github.com/jupyterhub/jupyterhub-idle-culler

File: py3-pycurl.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      - python-3-dev
+      - py3-setuptools
+      - curl-dev
+      - openssl-dev
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: 5730590be0271364a5bddd9e245c9cc0fb710c4cbacbdd95264a3122d23224ca
+      uri: https://files.pythonhosted.org/packages/a8/af/24d3acfa76b867dbd8f1166853c18eefc890fc5da03a48672b38ea77ddae/pycurl-${{package.version}}.tar.gz
```
Comment (mamccorm) [inline py3-pycurl.yaml:29]: Is there a reason we're not pulling this from git and polling for new tags?
- https://github.com/pycurl/pycurl

File: py3-sqlalchemy-cockroachdb.yaml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+      - busybox
+      - build-base
+      - python-3
+      - py3-setuptools
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: ddb46e2da53d812bf7b8ef21c3cafb918004747f815a283f374796bf57a9497a
+      uri: https://files.pythonhosted.org/packages/72/74/7157d36dee9175864f803fdbc931913dac17b0d4eee24727dbdac1244543/sqlalchemy-cockroachdb-${{package.version}}.tar.gz
```
Comment (mamccorm) [inline py3-sqlalchemy-cockroachdb.yaml:28]: Is there a reason we're not pulling this from git?
- https://github.com/cockroachdb/sqlalchemy-cockroachdb

I also noticed that the latest tag in Git is v2.0.0, which is strange - there doesn't seem to be a tag cut for 2.0.1

File: py3-jupyterhub-idle-culler.yaml (hunk 1 above)
Comment (mamccorm) [inline py3-jupyterhub-idle-culler.yaml:7]:
```suggestion
    - BSD-3-Clause
```

File: py3-jupyter-telemetry.yaml [hunk 5]
```diff
@@ -0,0 +1,41 @@
+# Generated from https://pypi.org/project/jupyter-telemetry/
+package:
+  name: py3-jupyter-telemetry
+  version: 0.1.0
+  epoch: 0
+  description: Jupyter telemetry library
+  copyright:
+    - license: BSD
```
Comment (mamccorm) [inline py3-jupyter-telemetry.yaml:8]: Re: https://github.com/jupyter/telemetry/blob/master/LICENSE

```suggestion
    - license: BSD-3-Clause
```

File: py3-certipy.yaml [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+      expected-sha256: 695704b7716b033375c9a1324d0d30f27110a28895c40151a90ec07ff1032859
+      uri: https://files.pythonhosted.org/packages/3a/b3/f9228354c1eac06cd3577a981571b9188392d73d583fcaa7a8f3fb374a56/certipy-${{package.version}}.tar.gz
+
+  - name: Python Build
+    uses: python/build-wheel
+
+  - uses: strip
+
+update:
+  enabled: false # doesn't exist in ReleaseMonitor, no releases or tags in GitHub, no new release since 2019.
```
Comment (mamccorm) [inline py3-certipy.yaml:36]: Are we sure this is not maintained? I did come across this Git repo, but i'm not sure if its the same application:
- https://github.com/ly4k/Certipy

Comment (Dentrax) [reply py3-certipy.yaml:36]: Yep, the corresponding repo is: https://github.com/LLNL/certipy

File: py3-sqlalchemy-cockroachdb.yaml (hunk 4 above)
Comment (Dentrax) [reply py3-sqlalchemy-cockroachdb.yaml:28]: Yep that's why I didn't use GitHub here.

**Review by mamccorm — APPROVED** [review]

**Pull request merged**